### ISSUE TITLE
Revive failing with annotation

### ISSUE THREAD

**Issue by ReedOnly** [morphy2k/revive-action#8]
With annotation turned on the Docker container fails and it seems like revive crashes.

Any idea?

![revive](https://user-images.githubusercontent.com/7236757/67390740-8285fb00-f59d-11e9-94e9-c6569b4a10d4.png)

**Comment by morphy2k** [comment]
Can you send me the log of the workflow run?

**Comment by ReedOnly** [comment]
```
2019-10-23T11:57:55.3761411Z ##[group]Run docker://morphy/revive-action:v1
2019-10-23T11:57:55.3761526Z with:
2019-10-23T11:57:55.3761607Z   config: .github/revive_config.toml
2019-10-23T11:57:55.3761691Z   path: api/...
2019-10-23T11:57:55.3761771Z env:
2019-10-23T11:57:55.3761930Z   GITHUB_TOKEN: ***
2019-10-23T11:57:55.3762012Z ##[endgroup]

2019-10-23T11:57:55.4029700Z ##[command]/usr/bin/docker run --name morphyreviveactionv1_d35cb7 --label fbc64f --workdir /github/workspace --rm -e GITHUB_TOKEN -e INPUT_CONFIG -e INPUT_PATH -e HOME -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e RUNNER_OS -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e GITHUB_ACTIONS=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/seismic-cloud/seismic-cloud":"/github/workspace" morphy/revive-action:v1

2019-10-23T11:57:59.7564922Z Successful run with 25 failures (25 warnings, 0 errors)
2019-10-23T11:58:00.1401914Z ##[error]Docker run failed with exit code 1
2019-10-23T11:58:00.1504441Z Cleaning up orphan processes
```

**Comment by morphy2k** [comment]
The annotations are displayed? If failures are found, this is the normal behavior. 
Maybe I should rethink this.

**Comment by ReedOnly** [comment]
No, annotations are not displayed, that's the problem, and docker container fails even when there should only be warnings...

**Comment by morphy2k** [comment]
The issue should be fixed with the latest release (v1.3). The annotation tool exited with code 1 on every failure regardless of severity.
